enhancements/two-nodes-openshift/2no.md

```diff
@@ -123,6 +127,26 @@ This functionality exists within RHEL-HA, but a wrapper will be provided to take
 
 When starting etcd, the OCF script will use etcd's cluster ID and version counter to determine whether the existing data directory can be reused, or must be erased before joining an active peer.
 
+
+### Summary of Changes
```

Comment:
Really good idea, I think this section is a great addition :+1:

Reply:
Thanks! :+1: